README.md

```diff
@@ -44,6 +44,12 @@ rm -rf package-lock.json
 ```
 
 ### USB connection
+First, make sure that your user is part of the `dialout` group if it is necessary in your Linux distribution. You can do this with the following command:
+```bash
+sudo usermod -a -G dialout youruser
+```
+where **youruser** is the user name you are running the program with.
```

Comment:
maybe, you can use something like `$USERNAME` where its value can be getting by using `whoami` in the terminal.